plugins/generator-1.18.2/forge-1.18.2/villagerprofession.definition.yaml

```diff
@@ -0,0 +1,14 @@
+templates:
+  - template: profession.mcmeta.ftl
+    writer: json
+    condition: isHatEnabled()
+    deleteWhenConditionFalse: true
```

Comment:
This parameter is redundant as of https://github.com/MCreator/MCreator/commit/5dbb97674d3097918f481ecd8cba62d9b3ba9040.

Reply:
Done!